weddineo-core/src/main/java/com/weddineo/api/controller/AuthenticationController.java

```diff
@@ -9,25 +10,29 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-@Api(value = "Authentication",
-        tags = { "Authentication" },
-        description = "API to handle weddineo authentication and authorization" )
+@Api(value = "Authentication", tags = {
+        "Authentication" }, description = "API to handle weddineo authentication and authorization")
 @RestController
 @RequestMapping("/auth")
 public class AuthenticationController {
     private RegistrationService registrationService;
+    private LogoutService logoutService;
 
-    public AuthenticationController(RegistrationService registrationService) {
+    public AuthenticationController(RegistrationService registrationService, LogoutService logoutService) {
         this.registrationService = registrationService;
+        this.logoutService = logoutService;
     }
 
-    @ApiOperation(
```

Comment:
we should configure our autoformatters on backend in the same way as soon as possible - we should discuss it on next meeting :) 
@PrzemyslawZnojek